### ISSUE TITLE
[NuGet] Contained dlls are not sent to output folder even when forced

### ISSUE THREAD

**Issue by Eideren** [xenko3d/xenko#513]
**Release Type**: Latest github

**Version**: https://github.com/xenko3d/xenko/commit/964ef5e68c4fa555d890f94c648e871f7e1bb067

**Platform(s)**: Any

**Describe the bug**
Some dlls, like ``Xenko.Games.dll``, are not copied to the output folder on build when being forced. It works fine for a simple application *locally* as .Net pulls it from the GAC or god knows where.

**To Reproduce**
1. Create a new dot net framework project.
2. Import nuget package Xenko.Games (here beta02-0674)
3. Add ``<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>`` within a ``PropertyGroup`` of your csproj
4. Reference that package within your main, ex:
```
static void Main(string[] args)
{
	System.Console.WriteLine(Xenko.Core.Platform.IsWindowsDesktop);
}
```
5. Build and look at the output, you should have a couple of dlls like ``Xenko.Shaders.dll``, ``Xenko.Core.Serialization`` and such but you won't have ``Xenko.Games.dll``.

**Expected behavior**
Dll -> output folder

**Screenshots**

**Log and callstacks**

**Additional context**

Adding ``<file src="*PATH_TO_THE_NETSTANDARD_DLL*\Xenko.Games.dll" target="lib\netstandard2.0\Xenko.Games.dll" />`` to the generated nuspec file resolves this issue.

**Comment by tebjan** [comment]
did you specify a runtime identifier in the toplevel exe project? line 'win-x64'? https://docs.microsoft.com/en-us/dotnet/core/rid-catalog

**Comment by Eideren** [comment]
Yes providing an rid does resolve the issue but I don't think it makes sense given those dlls are under standard.
Now that's just weird, after removing the rid the dlls still end up being included ? Perhaps a bug in nuget ?

**Comment by Eideren** [comment]
This issue has been fixed on latest master-3.1
